buildbot/master.cfg

```diff
@@ -347,6 +347,145 @@ def make_fifoci_win(type, mode="normal"):
+                            workdir="build/build",
+                            description="signing updater",
+                            descriptionDone="sign updater",
+                            haltOnFailure=True))
+
+    f.addStep(ShellCommand(command=["hdiutil", "create", "dolphin.dmg",
+                                    "-format", "UDBZ",
+                                    "-fs", "HFS+", # Needed for 7-Zip support
+                                    "-srcfolder", "dmg.dir", "-ov",
+                                    "-volname", "Dolphin"],
+                                    #"-volname", "dolphin.dmg"],
+                           workdir="build/build",
+                           logEnviron=False,
+                           description="packaging",
+                           descriptionDone="package"))
+
+    f.addStep(ShellCommand(command="/build/codesign.sh --deep dolphin.dmg",
+                           workdir="build/build",
+                           description="signing dmg",
+                           descriptionDone="sign dmg",
+                           haltOnFailure=True))
+
+    if mode == "normal":
+        master_filename = WithProperties("/srv/http/dl/builds/dolphin-%s-%s.dmg", "branchname", "shortrev")
+        url = WithProperties("https://dl.dolphin-emu.org/builds/dolphin-%s-%s.dmg", "branchname", "shortrev")
+    elif mode == "wip":
+        master_filename = WithProperties("/srv/http/dl/wips/%s-dolphin-%s-%s.dmg", "author", "branchname", "shortrev")
+        url = WithProperties("https://dl.dolphin-emu.org/wips/%s-dolphin-%s-%s.dmg", "author", "branchname", "shortrev")
+    elif mode == "pr":
+        master_filename = WithProperties("/srv/http/dl/prs/%s-dolphin-latest.dmg", "branchname")
+        url = WithProperties("https://dl.dolphin-emu.org/prs/%s-dolphin-latest.dmg", "branchname")
+    else:
+        master_filename = url = ""
+
+    master_filename, url = map(get_sharded_dl_path, (master_filename, url))
+
+    if master_filename and url:
+         f.addSteps(ReliableFileUpload(workersrc="build/dolphin.dmg",
+                                      masterdest=master_filename,
+                                      url=url, keepstamp=True, mode=0o644))
+
+    if mode == "normal":
+
+         f.addStep(MasterShellCommand(command="/home/buildbot/bin/send_build.py",
+                                        env={
+                                            "BRANCH": WithProperties("%s", "branchname"),
+                                            "SHORTREV": WithProperties("%s", "shortrev"),
+                                            "HASH": WithProperties("%s", "revision"),
+                                            "AUTHOR": WithProperties("%s", "author"),
+                                            "DESCRIPTION": WithProperties("%s", "description"),
+                                            "TARGET_SYSTEM": "macOS",
```

Comment:
Make this something like "macOS (universal)" so it shows up as a separate download option on the website until we've entirely phased out the non-universal builds.

Reply:
Does "USER_OS_MATCHER" need to be updated as well?

Reply:
Shouldn't need to, USER_OS_MATCHER is used to highlight builds to users on our download page. Roughly based on `USER_OS_MATCHER in headers['User-Agent'].lower()` basically.

Reply:
Okay, do you think it would be a good idea to update the original download button to say "macOS (Intel)" to help with clarity for users?

Reply:
More of a question for @mayimilae I think :-) Any thoughts?

Note that if we want to be consistent and have old/already built versions also show up as "macOS (Intel)" or whatever new string we decide on, I'll have to go and manually fixup the old builds data. Not a big deal, it's 5min of time, but I just need to be aware when that lands.

Reply:
Okay, I renamed the button to macOS (Intel) in the branch.